tests/tests/test_mtls.py

```diff
@@ -126,13 +126,15 @@ def hsm_get_key_uri(self, pin, ssl_engine_id, device):
             + '\nMODULE_PATH = /usr/lib/softhsm/libsofthsm2.so\ninit = 0\n" >> /etc/ssl/openssl.cnf'
         )
         device.run(
-            'sed -i.backup -e "/\\[Service\\]/ a Environment=SOFTHSM2_CONF=/softhsm/softhsm2.conf" /lib/systemd/system/mender-client.service'
+            'sed -i.backup -e "/\\[Service\\]/ a Environment=SOFTHSM2_CONF=/softhsm/softhsm2.conf" /lib/systemd/system/%s.service'
+            % client_service_name
         )
         return key_uri
 
     def hsm_cleanup(self, device):
         device.run(
-            "mv /lib/systemd/system/mender-client.service.backup /lib/systemd/system/mender-client.service"
+            "mv /lib/systemd/system/%s.service.backup /lib/systemd/system/%s.service"
+            % (client_service_name, client_service_name)
```

Comment:
Is the variable visible in these two locations?

Reply:
No, the test failed. Too quick search+replace late at night, sorry. I'll fix ASAP